dips/dip-10.md

```diff
@@ -151,9 +144,9 @@ The format of the command is:
 | Field 	    | Type 	     | Required? 	| Description 	           |
 |-------	    |------	     |-----------	|-------------	           |
 | _ObjectType   | str    | Y | Fixed value: `ReferenceIDCommand`|
-| sender        | str          | Y            | Sender's full Diem ID |
-| sender_address| str          | Y            | Sender's on-chain address |
-| recipient     | str          | Y            | Receiver's full Diem ID |
+| sender        | str          | Y            | Sender's full DiemID |
+| sender_address| str          | Y            | Sender's [bech32 identifier](https://github.com/diem/dip/blob/main/dips/dip-5.md) on-chain address |
```

Comment:
May consider use name "account identifier" instead of bech32 identifier, because it is defined in dip-5. Also need to mention subaddress should be None / 00000000